modules/cloudfront/distribution.tf

```diff
@@ -26,7 +26,11 @@ resource "aws_cloudfront_distribution" "main" {
     Name = var.tags_name
   }
 
-  logging_config = var.logging_config
+  logging_config {
+    bucket          = var.logging_config.bucket
+    prefix          = var.logging_config.prefix
+    include_cookies = var.logging_config.include_cookies
```

Comment:
it seems that in case when `var.logging_config` not passed so that it takes default value here we will have apply run failure as the default value is `{}`.
maybe we can solve this by using something like this: `bucket=try(var.logging_config.bucket, null)` ?
or by using variables optional() field-type describing ability available on terraform 1.3+ ? 
in any case there is need to test the cases